lib/fathead/node_js/output.txt

```diff
@@ -1078,24 +1029,15 @@ net Server	R	net.Server										http://nodejs.org/api/all.html#all_net server
 net.Server.address	A										<section class="prog__container"><p id='obj.desc'>Returns the bound address, the address family name, and port of the server\nas reported by the operating system.\nUseful to find which port was assigned when getting an OS-assigned address.\nReturns an object with <code>port</code>, <code>family</code>, and <code>address</code> properties:\n<code>{ port: 12346, family: &apos;IPv4&apos;, address: &apos;127.0.0.1&apos; }</code></p><p>Example:</p><pre><code class="lang-js">var server = net.createServer((socket) =&gt; {\n  socket.end(&apos;goodbye\n&apos;);\n}).on(&apos;error&apos;, (err) =&gt; {\n  // handle errors here\n  throw err;\n});\n\n// grab a random port.\nserver.listen(() =&gt; {\n  console.log(&apos;opened server on&apos;, server.address());\n});\n</code></pre><pre><code>server.address()</code></pre></section>	http://nodejs.org/api/all.html#all_net.server_address
 net.Server address	R	net.Server.address										http://nodejs.org/api/all.html#all_net.server_address
 net Server address	R	net.Server address										http://nodejs.org/api/all.html#all_net server_address
-net.Server.close	A										<section class="prog__container"><p id='obj.desc'>Stops the server from accepting new connections and keeps existing\nconnections. This function is asynchronous, the server is finally\nclosed when all connections are ended and the server emits a <code>&apos;close&apos;</code> event.\nThe optional <code>callback</code> will be called once the <code>&apos;close&apos;</code> event occurs. Unlike\nthat event, it will be called with an Error as its only argument if the server\nwas not open when it was closed.</p><pre><code>server.close([callback])</code></pre></section>	http://nodejs.org/api/all.html#all_net.server_close_callback
+net.Server.close	D								*[[net.Server.close(all_net.server_close)]] Emitted when the server closes. Note that if connections exist, this event is not emitted until all connections are ended.\n*[[net.Server.close(all_net.server_close_callback)]] Stops the server from accepting new connections and keeps existing connections. This function is asynchronous, the server is finally closed when all connections are ended and the server emits a &apos;close&apos; event. The optional callback will be called once the &apos;close&apos; event occurs. Unlike that event, it will be called with an Error as its only argument if the server was not open when it was closed.			
 net.Server close	R	net.Server.close										http://nodejs.org/api/all.html#all_net.server_close_callback
 net Server close	R	net.Server close										http://nodejs.org/api/all.html#all_net server_close_callback
```

Comment:
Hey @bfmags Sorry I didnt get you ? I ran the validation script on my side and I dont see any diffs.
Based on your gist if I compare the line numbers on the new output.txt generated after my changes I dont see the dup line numbers matching. Can you try running the validation script on the fresh output.txt may be ???